htsinfer/get_library_stats.py

```diff
@@ -18,8 +20,8 @@
 
 
 class GetLibStats:
-    """Determine library statitics of a single- or paired-end seguencing
-    library.
+    """Determine library statitics of a single- or paired-end \
+        sequencing library.
```

Comment:
Check PEP 8 and the Python style guide on docstrings (I think PEP 257)  to see how to format docstrings. Basically, if not everything fits on one line, the first line should contain only `"""`. Also, there should definitely not be any backslashes inside for wrapping lines.

Reply:
Fixed in commit.